app/views/admin/locations/_form.html.erb

```diff
@@ -13,15 +13,17 @@
 
   <div>
     <%= form.label :latitude, style: "display: block" %>
-    <%= form.text_field :latitude %>
+    <%= form.text_field :latitude, class: "form-control", id: "InputLatitude", placeholder: "Enter latitude" %>
```

Comment:
@lime666 write the `id` attribute using "kebab-case" like `id: 'input-latitude'`